main/urls.py

```diff
@@ -6,12 +6,16 @@
 app_name = 'main'
 
 urlpatterns = [
-    path('', views.BaseView.as_view(), name='base'),
-    path('home/', views.HomeView.as_view(), name='home'),
-    path('logIn/', views.LogInView.as_view(), name='login'),
-    path('logout/', views.log_out, name='logout'),
-    path('register/', views.SignUp.as_view(), name='register'),
-    path('home/SugJob/', views.SuggestionJobView.as_view(),name='SuggestionJob'),
-    path('profilepage/', views.UserProfileView.as_view(), name='detail'),
-    path('result', views.ResultView.as_view(), name='result'),
-] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
\ No newline at end of file
+                  path('', views.BaseView.as_view(), name='base'),
+                  path('home/', views.HomeView.as_view(), name='home'),
+                  path('logIn/', views.LogInView.as_view(), name='login'),
+                  path('logout/', views.log_out, name='logout'),
+                  path('register/', views.SignUp.as_view(), name='register'),
+                  path('home/SugJob/', views.SuggestionJobView.as_view(), name='SuggestionJob'),
```

Comment:
in the urls try to write the slug as 'home/suggestion-job' as SugJob does not indicate the purpose of it, and the name="suggestion_job", notice the 1st param is "-" and the last is "_"